helper/geojsonify.js

```diff
@@ -1,18 +1,17 @@
 
 var GeoJSON = require('geojson');
 var extent = require('geojson-extent');
-var labelGenerator = require('./labelGenerator');
 var logger = require('pelias-logger').get('api');
 var type_mapping = require('./type_mapping');
 var _ = require('lodash');
 var addDetails = require('./geojsonify_place_details');
 var addMetaData = require('./geojsonify_meta_data');
 
-function geojsonifyPlaces( params, docs ){
+function geojsonifyPlaces( params, docs, labelGenerator ){
```

Comment:
labelGenerator is not used throughout this file anymore as far as i can tell. should be removed from here and lines 14, 37